### PR TITLE
Generate formatted content for bot net block report

### LINKED ISSUE #8: Formatted email for BOT NET blocks
**Is your feature request related to a problem? Please describe.**
At the moment if a bot net block is found, the result from API as json will be sent to customers.

**Describe the solution you'd like**
We have received a sample result from CIRA so we can format the email and send only useful data.

**Describe alternatives you've considered**
N/A

**Additional context**
Sample result from CIRA:
```json
{
	"results": [
		{
			"time": 1603856700,
			"value": [
				{
					"name": "xpy8piady.ru",
					"threat-id": 1000,
					"count": 1,
					"first-occurrence": 1603856952897
				}
			]
		},
		{
			"time": 1604026800,
			"value": [
				{
					"name": "xpy8piady.ru",
					"threat-id": 1000,
					"count": 1,
					"first-occurrence": 1604027297651
				}
			]
		},
		{
			"time": 1603944900,
			"value": [
				{
					"name": "xpy8piady.ru",
					"threat-id": 1000,
					"count": 1,
					"first-occurrence": 1603945089994
				}
			]
		}
	],
	"isMax": false
}
```

### PR DESCRIPTION
**Pull request by ehsan-fj**
Fixes #8

### CODE REVIEW

File: defenz_alerts/main.py [hunk 1]
```diff
@@ -235,12 +235,13 @@ def generate_email_content(report_details, block_type, network, interval):
     return email_body
 
 
-def _generate_malware_phishing_email_content(report_details):
+def _generate_malware_web_email_content(report_details):
```
Comment (rafaelbernard) [inline defenz_alerts/main.py]: Maybe `_generate_malware_phishing_or_web_filter_email_content`?

File: defenz_alerts/main.py [hunk 2]
```diff
@@ -235,12 +235,13 @@ def generate_email_content(report_details, block_type, network, interval):
...
     """
-    Generates email body content for Malware Phishing block report.
+    Generates email body content for Malware Phishing and Web Filter
```
Comment (rafaelbernard) [inline defenz_alerts/main.py]:
```suggestion
    Generates email body content for Malware Phishing or Web Filter
```

File: defenz_alerts/main.py (hunk 1 above)
Comment (ehsan-fj) [reply defenz_alerts/main.py]: isn't that too long? I've put details in function doc string.

Comment (rafaelbernard) [reply defenz_alerts/main.py]: It is long, but much cleaner. The short version requires to read the body of the function to be sure what it does. I would say that sometimes it is OK to use the longer version in the sake of helping the human reader (as the machine does not care about the length anymore).

**Review by rafaelbernard — APPROVED** [review]

**Pull request merged**